examples/geneexpr_3tube.py

```diff
@@ -0,0 +1,52 @@
+# geneexpr.py - simple gene expression example
+# Richard M. Murray, 11 Aug 2018
+#
+# This example shows how to use the txtl library to create a model for
+# a simple gene expression construct.  This model is constructed to
+# demonstrate the ability to mimic the MATLAB TX-TL modeling toolbox
+# approach as well as a few simple variants that are enabled in
+# the python version.
+
+import txtl
+
+# Set up the standard TXTL tubes
+tube1 = txtl.extract('BL21_DE3')
+tube2 = txtl.buffer('stdbuffer')
+
+# Now set up a tube that will contain our DNA
+tube3 = txtl.newtube('geneexpr')
+
+# Define a DNA strand using strings (ala MATLAB)
+gene1 = txtl.assemble_dna('ptet(50)', 'BCD2(20)', 'tetR(1200)')
```

Comment:
We should take advantage of key=value pairs as function arguments in order to create a more readable code:
gene1 = txtl.assemble_dna(prom='ptet(50)', utr5='BCD2(20)', cds='tetR(1200)')
txtl.add_dna(mixture=tube3, dna=gene1, conc=1, type='plasmid')

Reply:
Added this variant to unit tests to confirm that it works.